Cargo.toml

```diff
@@ -39,9 +39,10 @@ default-features = false
 version = "0.2"
 
 [features]
-default = ["use_std"]
+default = ["use_std", "const_generics"]
 use_std = ["use_alloc", "either/use_std"]
 use_alloc = []
+const_generics = []
```

Comment:
Please don't include this.

Reply:
Oops, that wasn't supposed to be included, I thought I had it in a different branch

Reply:
Also realized I had made quite a few changes to that file for no net purpose, so I rebased the change out entirely.